src/hypercore-file-logger.js

```diff
@@ -61,103 +56,34 @@ class HyperCoreFileLogger extends HyperCoreLogger {
     this.republish = republish
     this.pathlike = pathlike
 
-    /** @type {Object<string, Tail>} */
-    this.fileTails = {}
-    /** @type {chokidar.FSWatcher | null} */
-    this.watcher = null
-    /** @type {boolean} */
-    this.isReady = false
+    this.watcher = new FilesWatcher(pathlike, republish, this.feedOpts.valueEncoding)
   }
 
   static getFileDelimiter () {
-    return ' >>> '
+    return FilesWatcher.getFileDelimiter()
   }
 
   static parseLine (line) {
```

Comment:
do we still need this method?

Reply:
https://github.com/bitfinexcom/hypercore-logs/pull/18#discussion_r840608709